sdks/python/scripts/run_integration_test.sh

```diff
@@ -143,8 +150,9 @@ if [[ -z $PIPELINE_OPTS ]]; then
   fi
 
   # Create a tarball if not exists
-  SDK_LOCATION=$(find ${SDK_LOCATION})
-  if [[ ! -f $SDK_LOCATION ]]; then
+  if [[ $(find ${SDK_LOCATION}) ]]; then
```

Comment:
I think this could be more simply written as the following, since `SDK_LOCATION` is no longer a glob pattern.
```sh
  if [[ ! -f ${SDK_LOCATION} ]]; then
    python setup.py -q sdist
    SDK_LOCATION="dist/apache-beam-*.tar.gz"
￼  fi
```

Reply:
`SDK_LOCATION` also hold value that passing from command-line flag `--sdk_location`. see [line 96](https://github.com/apache/beam/pull/6833/files/7632ca4e412696747730df801d55f9a3e58e6750#diff-03fc97650e97e71b4df5d7d3dc132355R96). This way, user can use glob in command-line flag as well.